compiler/rustc_codegen_llvm/src/gotoc/intrinsic.rs

```diff
@@ -304,6 +304,9 @@ impl<'tcx> GotocCtx<'tcx> {
             "atomic_xsub_relaxed" => codegen_atomic_binop!(sub),
             "breakpoint" => Stmt::skip(loc),
             "bswap" => self.codegen_expr_to_place(p, fargs.remove(0).bswap()),
+            // TODO: Handle new `caller_location` intrinsic
+            // https://github.com/model-checking/rmc/issues/374
+            "caller_location" => Stmt::skip(loc),
```

Comment:
Nit: should this be `codegen_unimplemented_intrinsic` since it isn't used in any tests?

Reply:
Good idea, thanks! Updated both the code and #375.